contracts/staking/cw20-stake-reward-distributor/src/contract.rs

```diff
@@ -34,19 +33,19 @@ pub fn instantiate(
     }
 
     let config = Config {
-        owner: owner.clone(),
         staking_addr: staking_addr.clone(),
         reward_token: reward_token.clone(),
         reward_rate: msg.reward_rate,
     };
     CONFIG.save(deps.storage, &config)?;
+    cw_ownable::initialize_owner(deps.storage, deps.api, &msg.owner)?;
```

Comment:
does the owner for existing contracts need to be initialized?

Reply:
oh great catch, thanks. i'll add migration logic for this as well